src/main/java/com/google/codeu/servlets/MessageServlet.java

```diff
@@ -34,53 +34,57 @@
 @WebServlet("/messages")
```

Comment:
Nit: keep whitespace to 2 spaces per indentation level, reference to [Google Style Guide](https://google.github.io/styleguide/javaguide.html#s4.2-block-indentation)